src/csharp/Grpc.Core/Server.cs

```diff
@@ -325,25 +344,41 @@ private void DisposeHandle()
             {
```

Comment:
Before it was a non-issue, but now I think if the handler throws then `AllowOneRpc` gets skipped? Should it the continuation be wrapped in `finally`?